oauth2client/transport/__init__.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transport is used to make HTTP requests."""
+
+from six.moves import http_client
+
+from oauth2client.transport import httplib2
+
+
+REFRESH_STATUS_CODES = (http_client.UNAUTHORIZED,)
+
+
+class _SETTINGS(object):
+    default_transport = httplib2
```

Comment:
I'm going to punt on this as I hope to remove it in a later commit, cool? If I don't end up removing it, you and @dhermes can figure which way is more appropriate.